src/modules/Builder/pages/BuilderApplet/BuilderApplet.schema.ts

```diff
@@ -2,24 +2,64 @@ import * as yup from 'yup';
 
 import i18n from 'i18n';
 import { getMaxLengthValidationError } from 'shared/utils';
-import { MAX_DESCRIPTION_LENGTH, MAX_NAME_LENGTH } from 'shared/consts';
+import {
+  ItemResponseType,
+  MAX_DESCRIPTION_LENGTH,
+  MAX_NAME_LENGTH,
+  MAX_SELECT_OPTION_TEXT_LENGTH,
+  MAX_SLIDER_LABEL_TEXT_LENGTH,
+} from 'shared/consts';
 
 const { t } = i18n;
 
 export const getIsRequiredValidateMessage = (field: string) =>
   t('validationMessages.isRequired', { field: t(field) });
 
-export const ItemSchema = () =>
-  yup
+export const ResponseValuesRowsSchema = () => ({
+  minLabel: yup.string().max(MAX_SLIDER_LABEL_TEXT_LENGTH, getMaxLengthValidationError),
+  maxLabel: yup.string().max(MAX_SLIDER_LABEL_TEXT_LENGTH, getMaxLengthValidationError),
+});
+
+export const ResponseValuesOptionsSchema = () =>
+  yup.array().of(
+    yup.object({
+      text: yup
+        .string()
+        .required(getIsRequiredValidateMessage('optionText'))
+        .max(MAX_SELECT_OPTION_TEXT_LENGTH, getMaxLengthValidationError),
+    }),
+  );
+
+export const ItemSchema = () => {
+  const itemSchema = yup
     .object({
-      name: yup.string().required(getIsRequiredValidateMessage('itemName')),
-      question: yup.string(),
+      name: yup
+        .string()
+        .required(getIsRequiredValidateMessage('itemName'))
+        .matches(/^[\w]+$/g, {
```

Comment:
I think it could be simplified to /^\w+$/g

Reply:
fixed